web_widget_x2many_2d_matrix/README.rst

```diff
@@ -50,15 +53,25 @@ field_label_y_axis
     Use another field to display in the table header
 field_value
     Show this field as value
+field_editability
+    Use this field to evaluate if a cell should be editable or not. It
```

Comment:
now reading your actual implementation, how about a dict that is simply merged into the attributes using `t-att="widget.get_xy_editability(x, y)"`? Then it might make sense to rename the field, but I envision a lot of cool crazy stuff people can do with this